.github/workflows/maven.yml

```diff
@@ -66,7 +66,7 @@ jobs:
         ./mvnw -V clean install -f ci.common --batch-mode --no-transfer-progress --errors -DtrimStackTrace=false -DskipTests
     # Run tests that require a minimum of Java 17 or later
     - name: Run tests that require a minimum of Java 17 or later
-      if: ${{ matrix.java == '17' || matrix.java == '21'}}
+      if: ${{ matrix.java == '17' || matrix.java == '21' || matrix.java == '25'}}
```

Comment:
Can we flip the check so we only do this when it is not Java 8 or 11? That way any new version of Java automatically will include these tests.